crates/biome_js_analyze/src/assists/source/sort_jsx_props.rs

```diff
@@ -0,0 +1,178 @@
+use std::{borrow::Cow, cmp::Ordering};
+
+use biome_analyze::{
+    context::RuleContext, declare_source_rule, ActionCategory, Ast, Rule, RuleAction, RuleSource,
+    RuleSourceKind, SourceActionKind,
+};
+use biome_console::markup;
+use biome_diagnostics::Applicability;
+use biome_js_factory::make::jsx_attribute_list;
+use biome_js_syntax::{AnyJsxAttribute, JsxAttribute, JsxAttributeList};
+use biome_rowan::{AstNode, BatchMutationExt};
+
+use crate::JsRuleAction;
+
+declare_source_rule! {
+    /// Enforce props sorting in JSX elements.
+    ///
+    /// This rule checks if the JSX props are sorted in a consistent way.
+    /// A spread prop resets the sorting order.
+    ///
+    /// The rule can be configured to sort props alphabetically, ignore case, and more.
```

Comment:
There's no more configuration, however we should document very well what's the order, meaning which kinds props come first and which ones come later.